### PR TITLE
Add project variable

### PR DESCRIPTION
**Pull request by GNewbury1**
# What this MR does
This MR allows specifying of which project `consul-cluster` will deploy to, by adding a project variable for each compute instance as `${var.gcp_project_id}`, as well as a project variable for the network and firewall rules, `${var.network_project_id}`.

# Why this MR is needed
The current method uses the project defined in the 'provider' block. This is good for single-project deployments, but when using `consul-cluster` with other terraform files which need deploying into other projects, there will be no project specified in `provider`. Each terraform object will have it's project specified instread. This MR will make sure that `consul-cluster` has the same ability.

### CODE REVIEW

File: modules/consul-cluster/variables.tf [hunk 1]
```diff
@@ -3,10 +3,19 @@
...
+
 variable "gcp_region" {
   description = "All GCP resources will be launched in this Region."
 }
 
+variable "gcp_zones" {
+  description = "All zones you want to deploy to within the specified region"
+  type = "list"
+}
+
```
Comment (josh-padnick) [inline modules/consul-cluster/variables.tf]: Why add this var? It doesn't appear to be used anywhere?

**Comment by GNewbury1** [comment]
I've changed `var.project` to `var.gcp_project_id` to have consistency with the other modules in this repository. I forgot I left `var.gcp_zones` in there, it was a change I was meaning to do for another branch to use `google_compute_region_instance_group_manager.distribution_policy_zones` to allow users to specify a list of zones to deploy within the given region, and single-zone deployments are no longer possible since the change from `google_compute_instance_group_manager` to `google_compute_region_instance_group_manager`

**Comment by josh-padnick** [comment]
Looks good. The only thing I now realize we're missing is an update to https://github.com/hashicorp/terraform-google-consul/blob/master/main.tf to include the new `project_id` variable. Once that happens, I think we're ready to merge. Thanks for this contribution!

**Comment by GNewbury1** [comment]
That should now be done

File: modules/consul-cluster/variables.tf (hunk 1 above)
Comment (GNewbury1) [reply modules/consul-cluster/variables.tf]: Removed in 19dd4d2

**Comment by GNewbury1** [comment]
## Major Change:

Similar to the change I made in https://github.com/hashicorp/terraform-google-vault/pull/25, I've just added a new variable called `network_project_id`. This allows the user to use a network which isn't in the project the Consul instances will be running.

**Comment by josh-padnick** [comment]
Merging now so I can validate with tests.

**Comment by josh-padnick** [comment]
Validated in #30. Once that's merged, I'll issue a new release!

**Pull request merged**